plugins/MultiSites/API.php

```diff
@@ -229,25 +237,11 @@ private function buildDataTable($sitesToProblablyAdd, $period, $date, $segment,
             }
         }
 
-        // get the data
-        // $dataTable instanceOf Set
-        $dataTable = $archive->getDataTableFromNumeric($fieldsToGet);
```

Comment:
Before we built a huge datable (map) and then merged them afterwards see `mergeChildren`, this was very expensive in terms of time.